.github/workflows/main.yml

```diff
@@ -44,6 +44,17 @@ jobs:
     
     -  name: Protobuf lint check
        run: ./buf check lint --input=model/src/main/proto --input-config buf.yaml && echo "Protobuf lint check completed successfully"
+
+    - name: Set up JDK 1.9
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.9
+
+    - name: XML lint check
+      run: ./gradlew lintDebug
```

Comment:
is this worth trying, as we are migrating to Bazel?
@BenHenning

Reply:
Is might help in covering Android Lint as per the issue here -  https://github.com/oppia/oppia-android/issues/1742

Reply:
Hmm, will this work if Gradle isn't defined anymore? What does Gradle call through to--could we call that ourselves such that we don't depend on Gradle?

Reply:
Android Lint without gradle is something I need to look. As of now I didn't see any other solution on running android lint without gradle.